fuzz/fuzz_targets/preserve_rdata.rs

```diff
@@ -18,6 +18,8 @@ use hickory_proto::{
     serialize::binary::{BinDecodable, BinEncodable},
 };
 
+type Result<T> = std::result::Result<T, &'static str>;
```

Comment:
I get that this is slightly more concise but I always feel like the indirection obscures the underlying types more than it helps understand what's going on.